pegasus/src/homepage.rb

```diff
@@ -327,156 +327,110 @@ def self.get_actions(request)
+        image: "/shared/images/courses/logo_tall_teacher2.jpg",
+        links:
+          [
+            {
+              text: "homepage_slot_text_link_elementary",
+              url: "/educate/curriculum/elementary-school"
+            },
+            {
+              text: "homepage_slot_text_link_middle",
+              url: "/educate/curriculum/middle-school"
+            },
+            {
+              text: "homepage_slot_text_link_high",
+              url: "/educate/curriculum/high-school"
+            }
+          ]
+      },
 
-        {
-          id: "advocate-en",
-          type: "block",
-          title: "homepage_slot_text_link_buy",
-          text: "homepage_slot_text_blurb_advocates",
-          color1: "185, 191, 21",
-          color2: "209, 213, 103",
-          url: "/help",
-          image: "/shared/images/courses/logo_tall_map.jpg",
-          links:
-            [
-              {
-                text: "homepage_slot_text_link_stats",
-                url: "/promote"
-              },
-              {
-                text: "homepage_slot_text_link_administrators",
-                url: "/yourschool"
-              },
-              {
-                text: "homepage_slot_text_link_donate",
-                url: "/donate"
-              }
-            ]
-        }
-      ].each {|entry| entry[:image].gsub!("/images/", "/images/fit-400/")}
-    else
-      [
-        {
-          id: "at-home-nonen",
-          type: "blockshort",
-          title: "homepage_slot_text_title_at_home",
-          text: "homepage_slot_text_blurb_at_home",
-          color1: "0, 173, 188",
-          color2: "89, 202, 211",
-          url: CDO.studio_url("/courses"),
-          image: "/images/mc/2016_homepage_hocblock.jpg"
-        },
-        {
-          id: "students-nonen",
-          type: "blockshort",
-          title: "homepage_slot_text_title_students",
-          text: "homepage_slot_text_blurb_students",
-          color1: "118, 101, 160",
-          color2: "166, 155, 193",
-          url: CDO.studio_url("/courses"),
-          image: "/shared/images/courses/logo_tall_elementary.jpg"
-        },
-        {
-          id: "educators-nonen",
-          type: "blockshort",
-          title: "homepage_slot_text_title_educators",
-          text: "homepage_slot_text_blurb_educators",
-          color1: "0, 148, 202",
-          color2: "89, 185, 220",
-          url: CDO.code_org_url("/teach"),
-          image: "/shared/images/courses/logo_tall_teacher2.jpg"
-        },
-        {
-          id: 'dance-nonen',
-          type: "blockshort",
-          title: 'studiobar_dance_title',
-          text: 'studiobar_dance_body',
-          color1: "185, 191, 21",
-          color2: "209, 213, 103",
-          url: '/dance',
-          image: '/shared/images/courses/logo_tall_dance.jpg'
-        }
-      ].each {|entry| entry[:image].gsub!("/images/", "/images/fit-400/")}
-    end
+      {
+        id: "advocate-en",
```

Comment:
This block is pretty US-centric -- are we sure we want to show it for all? (I'm generally onboard just want to double check)

Reply:
Yep, it's ok to show it for all! Here's a [spec doc](https://docs.google.com/document/d/1l17aLEKqW6vYBjaVKnbBbiYKnqfO-gsFX_tFIzNLZUE/edit#heading=h.qi8xxyq72bct) with more details if you're curious. We're also planning on redesigning the homepage early next year so this section will become obsolete.